java-checks/src/main/java/org/sonar/java/checks/DeadStoreCheck.java

```diff
@@ -127,6 +128,9 @@ private void checkElements(CFG.Block block, Set<Symbol> blockOut, Symbol.MethodS
           break;
         case TRY_STATEMENT:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 8 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1151)